sdk/core/ts-http-runtime/src/util/bytesEncoding.ts

```diff
@@ -21,5 +36,96 @@ export function uint8ArrayToString(bytes: Uint8Array, format: EncodingType): str
  * @returns a uint8array
  */
 export function stringToUint8Array(value: string, format: EncodingType): Uint8Array {
-  return Buffer.from(value, format);
```

Comment:
I do wonder about the perf implications of doing our browser version here. @timovv any thoughts?

Reply:
Appears we are not making this change in `core-util` so not sure why we are doing it here? I do think we should keep using the Node primitives here if possible (the browser version is more of a polyfill and I agree there are perf implications to consider). At the very least we should be consistent across packages.

Reply:
This was why it was kept in, is that there may be a plan to speed up the perf of those.  See https://github.com/nodejs/node/pull/38433#issuecomment-828426932

Reply:
Closing for now, but may revisit.